intel/docs/Meshery.md

```diff
@@ -0,0 +1,147 @@
+# Use Meshery to deploy and manage Istio with Intel Features
+
+## Meshery Introduction
+
+Meshery is the open source, cloud native management plane that enables the adoption, operation, and management of Kubernetes, any service mesh, and their workloads.
+
+### The benefit to use Meshery to deploy and manage Istio
+
+Meshery offers us with an intuitive, visual, and convenient way to deploy Intel Accelerated Istio with one button and run performance tests to intuitively demonstrate the performance acceleration of Intel Istio.
+
+## Architecture
+
+<img src="./images/Architecture.png" width="600px">
+
+## Install Meshery
+
+**Install on Kubernetes**:
+
+```
+$ curl -L https://meshery.io/install | PLATFORM=kubernetes bash -
+```
+
+**Install on Docker**:
+
+```
+$ curl -L https://meshery.io/install | PLATFORM=docker bash -  
+```
+ 
+**Install via Docker Compose**:
+
+If you want more personalized configurations, such as setting proxies, please edit “.meshery/meshery.yaml”, then setup Meshery via docker compose:
+
+```
+$ docker compose -f ~/.meshery/meshery.yaml up
+```
+
+## Access and Login Meshery
+
+Visit Meshery’s web-based user interface http://<hostname>:9081. 
+
+<img src="./images/acess-meshery.png" width="600px">
+
+## Configure Connection to Kubernetes
+
+Meshery attempts to auto detect your kubeconfig if it is stored in the default path ($HOME/.kube) on your system. In most deployments, Meshery will automatically connect to your Kubernetes cluster. If your config has not been auto-detected, or you want to connect remote Kubernetes cluster, you can manually locate and upload your kube config file and select the context name (docker-desktop, kind-clsuter, minikube etc.)
+
+<img src="./images/connect-k8s-cluster.png" width="600px">
+
+## Meshery Design 
+
+Meshery Designs contain patterns and configurations that describe how we will deploy Istio.
+Currently, we have published two Meshery Designs in Meshery Catalog website which you can import and deploy using Meshery directly:
+- [CRYPTOMB-TLS-HANDSHAKE-ACCELERATION-FOR-ISTIO](https://raw.githubusercontent.com/meshery/meshery.io/master/catalog/28715e69-c6c1-4f96-bfa2-05113b00bae0.yaml)
+- [QAT-TLS-HANDSHAKE-ACCELERATION-FOR-ISTIO](https://raw.githubusercontent.com/meshery/meshery.io/master/catalog/05e97933-90a6-4dd3-9b29-18e78eb4d3f1.yaml)
+
```

Comment:
possible to add some pre-req for this? Like enable QAT device plugin, etc.

Reply:
In fact, QAT Design itself installs the QAT Device plugin. But it would be helpful to add some descriptions of what these designs do, so I added them.